tests/installation/module_registration/register_module_regpage.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: After register Application module
+#          in "Extension and Module Selection" dialog
+#          has register page
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use registration;
+use testapi qw(save_screenshot get_var get_required_var);
+
+our %ADDONS_REGCODE = (
+    'sle-ha' => get_var('SCC_REGCODE_HA'),
+    'sle-ha-geo' => get_var('SCC_REGCODE_GEO'),
```

Comment:
I'm not sure if it is necessary at this point to gather all the possible regcodes, we are using libyui-rest-api only for sle-15-sp{3,4}, perhaps it is better to not give false hopes that this would work and we can do this later on demand.  we just need 'we' for now? right? a hash with one for now?

Reply:
Do we need live patching to do register at here?